### PR TITLE
Adding hadoop dependency for azure

### PR DESCRIPTION
**Pull request by CuriousVini**
Issue: https://issues.cask.co/browse/CDAP-11599
Build: https://builds.cask.co/browse/CDAP-RUT1008-1

Did `mvn clean install -DskipTests` on cdap repo and ran `mvn clean test -P standalone-test` in integration test repo. 

image

image

### CODE REVIEW

File: cdap-standalone/pom.xml [hunk 1]
```diff
@@ -237,6 +237,16 @@
...
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-azure</artifactId>
+      <version>${sdk.hadoop.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-storage</artifactId>
+      <version>2.2.0</version>
```
Comment (YanAtCask) [inline cdap-standalone/pom.xml:248]: Just wonder what to make choose this version, not 5.*, even 4.*?

Comment (CuriousVini) [reply cdap-standalone/pom.xml:248]: This is because we were getting contentmd5 header missing exceptions with higher version

**Comment by CuriousVini** [comment]
Build is green.

**Comment by yaojiefeng** [comment]
Can we run standalone ITN on this branch and make sure this will not break ITN?

**Comment by yaojiefeng** [comment]
LGTM other than that

**Comment by CuriousVini** [comment]
ITN passes on local. Merging.

**Pull request merged**